airflow/models/dagbag.py

```diff
@@ -350,7 +350,7 @@ def parse(mod_name, filepath):
                 sys.modules[spec.name] = new_module
                 loader.exec_module(new_module)
                 return [new_module]
-            except Exception as e:
+            except (Exception, BaseException) as e:
```

Comment:
```suggestion
            except (Exception, AirflowTaskTimeout) as e:
```

Reply:
If we want to catch AirflowTaskTimeout, then we should use it here. Catching the BaseException isn't a best practice. But if we are sure that we want to catch everything, then we could just leave `except BaseException as e`:

Reply:
thank you for the idea :)